src/Microsoft.Azure.NotificationHubs/NotificationOutcome.cs

```diff
@@ -30,32 +30,44 @@ public sealed class NotificationOutcome
         public NotificationOutcomeState State { get; internal set; }
 
         /// <summary>
-        /// Gets or sets the success value of the outcome.
+        /// Gets or sets a number of devices that successfully received the notification.
         /// </summary>
         /// 
         /// <returns>
-        /// The success value of the outcome.
+        /// A number of devices that successfully received the notification.
```

Comment:
This is a step forward, as the language is much more specific. I don't like "a number" in either this or the above though. It makes it sound like there's still room for play. Please alter to the "the number".